### PR TITLE
Add tags to services

### PR DESCRIPTION
**Pull request by havan**
This PR adds service tags according to the [service tags spec](https://github.com/chain4travel/camino-messenger-protocol/blob/dev/spec/service-tags.md) to all of the defined services.

### CODE REVIEW

File: proto/cmp/services/book/v1/validate.proto [hunk 1]
```diff
@@ -48,6 +48,8 @@ message ValidationObject {
 // ![Diagram](https://storage.googleapis.com/docs-cmp-files/diagrams/proto/cmp/services/book/v1/validate.proto.dot.xs.svg)
 //
 // [Open Message Diagram](https://storage.googleapis.com/docs-cmp-files/diagrams/proto/cmp/services/book/v1/validate.proto.dot.svg)
+//
+/// @custom:cmp-service type:product routing:p2p on-chain:true
```
Comment (SamJaarsma) [inline proto/cmp/services/book/v1/validate.proto]: on-chain is false for validate

File: proto/cmp/services/book/v2/validate.proto [hunk 2]
```diff
@@ -48,6 +48,8 @@ message ValidationObject {
 // ![Diagram](https://storage.googleapis.com/docs-cmp-files/diagrams/proto/cmp/services/book/v2/validate.proto.dot.xs.svg)
 //
 // [Open Message Diagram](https://storage.googleapis.com/docs-cmp-files/diagrams/proto/cmp/services/book/v2/validate.proto.dot.svg)
+//
+/// @custom:cmp-service type:product routing:p2p on-chain:true
```
Comment (SamJaarsma) [inline proto/cmp/services/book/v2/validate.proto]: on-chaine is false for validate

File: proto/cmp/services/book/v3/validate.proto [hunk 3]
```diff
@@ -48,6 +48,8 @@ message ValidationObject {
 // ![Diagram](https://storage.googleapis.com/docs-cmp-files/diagrams/proto/cmp/services/book/v3/validate.proto.dot.xs.svg)
 //
 // [Open Message Diagram](https://storage.googleapis.com/docs-cmp-files/diagrams/proto/cmp/services/book/v3/validate.proto.dot.svg)
+//
+/// @custom:cmp-service type:product routing:p2p on-chain:true
```
Comment (SamJaarsma) [inline proto/cmp/services/book/v3/validate.proto]: on-chain is false for validate

File: proto/cmp/services/partner/v1/partner_configuration.proto [hunk 4]
```diff
@@ -35,6 +35,8 @@ message GetPartnerConfigurationResponse {
 // ![Diagram](https://storage.googleapis.com/docs-cmp-files/diagrams/proto/cmp/services/partner/v1/partner_configuration.proto.dot.xs.svg)
 //
 // [Open Message Diagram](https://storage.googleapis.com/docs-cmp-files/diagrams/proto/cmp/services/partner/v1/partner_configuration.proto.dot.svg)
+//
+/// @custom:cmp-service type:core routing:local on-chain:true
```
Comment (SamJaarsma) [inline proto/cmp/services/partner/v1/partner_configuration.proto]: not sure if this is type:core or type:service

File: proto/cmp/services/partner/v2/partner_configuration.proto [hunk 5]
```diff
@@ -35,6 +35,8 @@ message GetPartnerConfigurationResponse {
 // ![Diagram](https://storage.googleapis.com/docs-cmp-files/diagrams/proto/cmp/services/partner/v2/partner_configuration.proto.dot.xs.svg)
 //
 // [Open Message Diagram](https://storage.googleapis.com/docs-cmp-files/diagrams/proto/cmp/services/partner/v2/partner_configuration.proto.dot.svg)
+//
+/// @custom:cmp-service type:core routing:local on-chain:true
```
Comment (SamJaarsma) [inline proto/cmp/services/partner/v2/partner_configuration.proto]: not sure if this is type:core or type:service

File: proto/cmp/services/partner/v1/partner_configuration.proto (hunk 4 above)
Comment (havan) [reply proto/cmp/services/partner/v1/partner_configuration.proto]: We don't have `type:service`. Do you mean `type:system`?

File: proto/cmp/services/partner/v2/partner_configuration.proto (hunk 5 above)
Comment (havan) [reply proto/cmp/services/partner/v2/partner_configuration.proto]: We don't have `type:service`. Do you mean `type:system`?

File: proto/cmp/services/partner/v1/partner_configuration.proto (hunk 4 above)
Comment (SamJaarsma) [reply proto/cmp/services/partner/v1/partner_configuration.proto]: sorry I meant type:product

File: proto/cmp/services/partner/v2/partner_configuration.proto (hunk 5 above)
Comment (SamJaarsma) [reply proto/cmp/services/partner/v2/partner_configuration.proto]: I meant type:product

File: proto/cmp/services/book/v3/validate.proto (hunk 3 above)
Comment (havan) [reply proto/cmp/services/book/v3/validate.proto]: Fixed

File: proto/cmp/services/book/v2/validate.proto (hunk 2 above)
Comment (havan) [reply proto/cmp/services/book/v2/validate.proto]: Fixed

File: proto/cmp/services/book/v1/validate.proto (hunk 1 above)
Comment (havan) [reply proto/cmp/services/book/v1/validate.proto]: Fixed

File: proto/cmp/services/partner/v1/partner_configuration.proto (hunk 4 above)
Comment (havan) [reply proto/cmp/services/partner/v1/partner_configuration.proto]: Switched to `type:product`

File: proto/cmp/services/partner/v2/partner_configuration.proto (hunk 5 above)
Comment (havan) [reply proto/cmp/services/partner/v2/partner_configuration.proto]: Switched to `type:product`

**Review by SamJaarsma — APPROVED** [review]
LGTM

**Pull request merged**